src/scripts/CMSSync/settings.ts

```diff
@@ -0,0 +1,20 @@
+import { SettingsObject, SettingsType } from './types'
+
+const settings: SettingsObject[] = [
+  {
+    name: 'ifos',
+    link: 'https://pancake-config-api-chefkai.pancakeswap.vercel.app/ifos',
```

Comment:
`https://pancake-config-api-chefkai.pancakeswap.vercel.app` should be in a variable.

Reply:
Also, rename link in url. Link is a frontend entity.

Reply:
Do you want to add this to an env variable?

Reply:
Both are ok